master/internal/config/resource_config.go

```diff
@@ -1,4 +1,4 @@
-package resourcemanagers
```

Comment:
Bradley made a suggesting that config should perhaps live in master/pkg instead of master/internal, but I don't think that's a change we need to make now. Just an FYI.

Reply:
That's a really good point, I'll make a note to do this in the future.